strings/slices/slices.go

```diff
@@ -20,21 +20,14 @@ limitations under the License.
 // replace "stringslices" if the "slices" package becomes standard.
 package slices
 
+import "slices"
```

Comment:
I think this should alias the import to "goslices" or "stdslices" for clarity?

Reply:
Changed to "goslices", thanks.